examples/config files - basic/3 connector-ldap.yml

```diff
@@ -30,7 +30,7 @@ credential_manager:
     type: windows_credential_manager
     #service_name is Required for Windows Credential Manger
     service_name: "Internet or Network Address field in Credential Manager"
-    username: "UserName"
+    service_username: "UserName"
```

Comment:
No, you missed the point of my earlier comment, sorry if I was unclear.  I am saying that, if the customer doesn't mind having the `username` field in plain text in this file, then you should just *remove* the separate specification of username in this section and use the `username` setting that is already specified in the LDAP config.  All you need in this section is the `service_name`.

What confused you might be that I also considered a *separate* use case: If the customer is nervous about even including the username as plain text in the config file, then you could fetch both the username and the password from credential storage.  In that case you have to either have to store a "password" for the `service_name` which is the concatenation of the actual username and password (with some concatenation character), _or_ have two "service name" keys in this section: one that stored the "username" as its password, and one that stored the "password" as its password.  Does that help?